src/components/sbb-journey-summary/readme.md

```diff
@@ -0,0 +1,34 @@
+# sbb-journey-summary
+
```

Comment:
You can add here some explanation for the component:

- How we would consume it? (markup)
- How many slots can we add?